tests/test_config.py

```diff
@@ -0,0 +1,102 @@
+
+import os
+import sys
+import git
+import glob
+import pytest
+
+import csv
+import argparse
+from argparse import Namespace
+import yaml
+from catalogue.config import config, config_validator
+
+
+
+
+def test_no_config(test_args, capsys, tmpdir):
+
+    os.chdir(tmpdir)
+    config(test_args)
+    captured = capsys.readouterr()
+    assert "No previous config file found" in captured.out
+
+
+
+def test_existing_config(test_args, capsys, tmpdir):
+
+    # With existing config file
+    output_dir = os.path.join(tmpdir, 'results')
+    input_dir = os.path.join(tmpdir, 'data')
+    setattr(test_args, "output_data", output_dir)
+    os.chdir(tmpdir)
+
+    #TODO: manually create something (e.g. a fixture) instead of just running it twice
```

Comment:
Not sure you need a fixture here if this is the only test that uses this object. If you were to write a new test that does use this, then I would make a fixture.

Reply:
Do you mean the `test_args` fixture? I think this one was created beforehand.